test/e2e/specs/all-features-confirmation/errors/errors.spec.js

```diff
@@ -310,7 +310,7 @@ describe('Error related test cases,', function() {
                 var newapplink = url.replace("recordedit", "recordset"),
                     lastSlash = newapplink.lastIndexOf("/"),
                     recordsetUrl = newapplink.slice(0, lastSlash);
-                expect(currentUrl).toBe(recordsetUrl, "The redirection from record page to recordset/search in case of multiple records failed");
+                expect(currentUrl).toContain(recordsetUrl, "The redirection from record page to recordset/search in case of multiple records failed");
```

Comment:
This test case always failed. A similar test case for the Record App passes because we extract the part of the url before the last '/ 'and then compare it with the expected url.

Reply:
Is `recordsetUrl` not actually a url? but actually just the `id=<some-value>`?

Reply:
It is the first part of the url until the table name, without any modifiers.